src/BlueSCSI.cpp

```diff
@@ -538,6 +551,14 @@ void findDriveImages(FsFile root) {
       file_test.close();
       break;
     }
+
+    // Look for this file to enable MSTE_MODE
+    // hacky until a better ini file parsing exists
+    if(strncmp(name, "MSTE_MODE", 9) == 0) {
+      megaste_mode = true;
```

Comment:
Lets add a log statement to the file just so when we're debugging and someone provides us a log file we'll know. `MSTE_MODE: On` or similar.